### PR TITLE
docs: add leaky styles to the faq

### PR DESCRIPTION
**Pull request by thepassle**

### CODE REVIEW

File: docs/faq/README.md [hunk 1]
```diff
@@ -123,3 +123,46 @@ ReferenceError: process is not defined
...
+  --owc-blue: #217ff9;
+}
+
+/* use inside your component: */
+:host {
+  background-color: var(--owc-blue);
+}
+```
+
+If this inheriting behavior is undesirable, you can reset it by adding the following CSS to your component:
```
[the remark this answers is not in the document]
Comment (thepassle) [reply docs/faq/README.md]: updated

File: docs/.vuepress/public/demoing/demo/custom-elements.json [hunk 2]
```diff
@@ -47,13 +47,18 @@
...
         {
-          "name": "--demo-wc-card-back-color",
-          "description": "Font color for the back",
+          "name": "--demo-wc-card-header-color-front",
+          "description": "Header color for the front",
+          "type": "Color"
+        },
+        {
+          "name": "--demo-wc-card-header-color-back",
+          "description": "Header color for the back",
```
Comment (thepassle) [inline docs/.vuepress/public/demoing/demo/custom-elements.json]: errr not sure how this ended up in here

File: docs/faq/README.md [hunk 3]
```diff
@@ -123,3 +123,50 @@ ReferenceError: process is not defined
...
+
+## Why do certain styles pierce the shadow DOM?
+
+> "Doesn't shadow DOM provide total encapsulation?"
+
+Nope! As written in the [spec](https://www.w3.org/TR/css-scoping-1/#inheritance):
+
+> The top-level elements of a shadow tree inherit from their host element.
+
+What this means is that [_inheritable_](https://gist.github.com/dcneiner/1137601) styles continue to inherit in shadow DOM, will _pierce_ the shadow DOM and affect your component's styling.
```
Comment (daKmoR) [inline docs/faq/README.md]: can we add the most common use cases people will run into here?
something like 

"this includes for example color or font"

Comment (thepassle) [reply docs/faq/README.md]: done

File: docs/.vuepress/public/demoing/demo/custom-elements.json (hunk 2 above)
Comment (thepassle) [reply docs/.vuepress/public/demoing/demo/custom-elements.json]: done

**Comment by daKmoR** [comment]
nice 👍

**Pull request merged**